### PR TITLE
Add initial support for Graylog2 through graypy

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -21,8 +21,10 @@ def read(filename):
     'requests',
     'feedparser',
     'pytz',
+    'google-api-python-client',
```
Comment (imankulov) [inline setup.py]: The result of an invalid merge, most likely :)

**Comment by imankulov** [comment]
@eleftherios-c you tested how it will work if graylog is not configured?
1. Looks like the `env` configuration won't work without environment variables
2. What if we initialize the graylog client with empty values, will it quietly skip logging requests?

The env is a bit tricky, and doesn't work as you would probably expect

``` python
>>> import environ
>>> env = environ.Env()
>>> env('FOO', 'default_foo')                                                                                                                         
Traceback (most recent call last):
...
django.core.exceptions.ImproperlyConfigured: Set the FOO environment variable
```

Here is the way to go

``` python
>>> env = environ.Env(FOO=('str', 'default_foo'))
>>> env('FOO')
'default_foo'
```

File: setup.py (hunk 1 above)
Comment (ghost) [inline setup.py]: But the 2 commits ([the one adding the support](https://github.com/Doist/powerapp/commit/8b8e629bbef3e78dc6ffa80823e7db87f5269402#diff-2eeaed663bd0d25b7e608891384b7298) and [the merge with master](https://github.com/Doist/powerapp/commit/62803535ff98aad2e7620b9b3b3bb73744cfac95#diff-2eeaed663bd0d25b7e608891384b7298) don't contain and line like that, that is strange...

Was that like removed by you, and readded by the merge?

**Comment by ghost** [comment]
I tested that when the Graylog2 host is set to an empty string `''`, then no error is produced and this logging handler seems not to be used.  For example when it is set to `None` an error is produced by `graypy`.

I'll refactor the env handling.

**Comment by ghost** [comment]
I refactored the inclusion of Graylog2 in the Django settings, and I think the problem with the setup.py was also fixed (by mistake a requirement removed was still kept back).

Now, we should not worry how graypy is skipped or not, since nothing is added to Django's config if GRAYLOG2_HOST is not set.

**Pull request merged**